addon/mixins/ember-filepicker.js

```diff
@@ -24,28 +24,30 @@ export default Ember.Mixin.create({
 	onSelection: null,
 	onError: null,
 	onClose: null,
-	options : {},
+        pickerOptions : {},
+        storeOptions : {},
 	filepicker: Ember.inject ? Ember.inject.service() : null,
 	openFilepicker: function() {
 		Ember.run.scheduleOnce('afterRender', this, function(){
 			this.get('filepicker.promise').then(Ember.run.bind(this, function(filepicker) {
-				var options = this.get('options');
-        if (options && options.useStore) {
-            filepicker.pickAndStore(
-              options.picker,
-              options.store,
-              Ember.run.bind(this, this.handleSelection),
-              Ember.run.bind(this, this.handleError)
-            );
-        }
-        else {
-          filepicker.pick(
-            options.picker,
-            Ember.run.bind(this, this.handleSelection),
-            Ember.run.bind(this, this.handleError)
-          );
-        }
-			}));
-		});
-	}.on('didInsertElement')
+                                var pickerOptions = this.get('pickerOptions');
+                                var storeOptions = this.get('storeOptions');
+                                if (pickerOptions && storeOptions) {
+                                        filepicker.pickAndStore(
+                                            pickerOptions,
+                                            storeOptions,
+                                            Ember.run.bind(this, this.handleSelection),
+                                            Ember.run.bind(this, this.handleError)
+                                        );
+                                }
+                                else {
+                                    filepicker.pick(
+                                        pickerOptions,
+                                        Ember.run.bind(this, this.handleSelection),
+                                        Ember.run.bind(this, this.handleError)
+                                    );
+                                }
+                        }));
+                });
+        }.on('didInsertElement')
```

Comment:
I completely agree.  I don't know why it was screwy in my editor - this was me actually trying to fix it.  I've updated it properly in my repo now - how I do I get the updates into this pull request?